docs/sources/alerting/alerting-rules/_index.md

```diff
@@ -12,37 +12,54 @@ labels:
+      destination: /docs/grafana/<GRAFANA_VERSION>/alerting/alerting-rules/create-grafana-managed-rule/
+    - pattern: /docs/grafana-cloud/
+      destination: /docs/grafana-cloud/alerting-and-irm/alerting/alerting-rules/create-grafana-managed-rule/
+  configure-ds-alerts:
+    - pattern: /docs/grafana/
+      destination: /docs/grafana/<GRAFANA_VERSION>/alerting/alerting-rules/create-data-source-managed-rule/
+    - pattern: /docs/grafana-cloud/
+      destination: /docs/grafana-cloud/alerting-and-irm/alerting/alerting-rules/create-data-source-managed-rule/
+  recording-rules:
+    - pattern: /docs/grafana/
+      destination: /docs/grafana/<GRAFANA_VERSION>/alerting/alerting-rules/create-recording-rules/
+    - pattern: /docs/grafana-cloud/
+      destination: /docs/grafana-cloud/alerting-and-irm/alerting/alerting-rules/create-recording-rules/
+  alert-types-comparison-table:
+    - pattern: /docs/grafana/
+      destination: /docs/grafana/<GRAFANA_VERSION>/alerting/fundamentals/alert-rules/#comparison-between-alert-rule-types
+    - pattern: /docs/grafana-cloud/
+      destination: /docs/grafana-cloud/alerting-and-irm/alerting/fundamentals/alert-rules/#comparison-between-alert-rule-types
 ---
 
 # Configure alert rules
 
-An alert rule consists of one or more queries and expressions that select the data you want to measure. It also contains a condition, which is the threshold that an alert rule must meet or exceed in order to fire.
+[Alert rules](ref:alert-rules) are the central component of your alerting system.
 
-Create, manage, view, and adjust alert rules to alert on your metrics data or log entries from multiple data sources — no matter where your data is stored.
+An alert rule consists of one or more queries and expressions that select the data you want to measure. It contains a condition to trigger the alert, an evaluation period that determines how often the rule is evaluated, and additional options to manage alert events and their notifications.
 
-The main parts of alert rule creation are:
+Grafana supports two types of alert rules:
 
-1. Select your data source
-1. Query your data
-1. Normalize your data
-1. Set your threshold
+1. Grafana-managed alert rules: These can query multiple data sources.
 
-**Query, expressions, and alert condition**
+1. Data source-managed alert rules: These can only query Prometheus-based data sources and support horizontal scaling.
 
-What are you monitoring? How are you measuring it?
+We recommend using Grafana-managed alert rules whenever possible, and opting for data source-managed alert rules when horizontal scaling is required. Refer to the [comparison table of alert rule types](ref:alert-types-comparison-table) for a more detailed overview.
 
-{{< admonition type="note" >}}
-Expressions can only be used for Grafana-managed alert rules.
-{{< /admonition >}}
+Both types of alert rules can be configured in Grafana using the **+ New alert rule** flow . For step-by-step instructions, refer to:
```

Comment:
Both types of alert rules can be configured in Grafana using the **+ New alert rule** flow. For step-by-step instructions, refer to: